### DISCUSSION TITLE
Publish Resource Manager module earlier than New Version of DNN

### DISCUSSION THREAD

**Discussion by mortezakermani** [dnnsoftware/Dnn.Platform#5500] [General]
Hi,
The Resource Manager module in DNN 9.11.00 had some major issues and I know some of them are fixed and the module will be available in next release of DNN but here I'm asking you to give us an option to download last version of this module earlier than new release of DNN.
Our websites and our users have troubles with this module and it really sucks.

Thanks

**Comment by valadas** [comment]
It is not advisable since it has not passed any RC stage. However, if you are a risk taker and want to try it, you could get the latest successful build from the develop branch here https://artprodwus21.artifacts.visualstudio.com/Ad394b248-b709-4c8a-9adc-6ed9de8c8163/9defc7c7-d48d-4a8b-9f46-fd11ba3c5aad/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL2RvdG5ldC9wcm9qZWN0SWQvOWRlZmM3YzctZDQ4ZC00YThiLTlmNDYtZmQxMWJhM2M1YWFkL2J1aWxkSWQvODYxMzQvYXJ0aWZhY3ROYW1lL0FydGlmYWN0czE1/content?format=zip

But do not install the complete DNN, extract the install or upgrade zip and inside its install/module folder you will find a package just for the resource manager. You could possibly be able to install that on your existing instance.

Again this is not a good advice, just something one could do. Be sure to backup first just to be on the safe side...

**Reply by mortezakermani** [reply]
I really appreciate your help Valadas. 

I am DNN expert and I'll take your advice so don't worry about it.

Thanks again.

**Comment by valadas** [comment]
And, if nothing crazy comes up, 9.11.1RC might go out in a couple of days (not a guarantee) and since it is a patch release, the minimal RC time is just 1 week... We could find stuff during RC though and this is also not a 1week guarantee, just the best case scenario...
